examples/ThreeD_TaylorGreenVortex.jl

```diff
@@ -24,4 +24,4 @@ function ω_mag_data(sim)
     return @view sim.flow.σ[2:end-1,2:end-1,2:end-1]
 end
 
-sim,fig = volume_plot(TGV(),ω_mag_data,fname="TGV.mp4",duration=10)
+sim,fig = volume_video!(TGV(),ω_mag_data,name="TGV.mp4",duration=10)
```

Comment:
what's this change?

Reply:
sorry, this change landed in this commit, went too fast. I couldn't find a function named `volume_plot` in the project, I assumed `volume_video!` was meant instead.

Reply:
ok.